src/main/java/uk/gov/hmcts/cmc/claimstore/services/CountyCourtJudgmentService.java

```diff
@@ -58,6 +66,13 @@ public Claim save(
 
         countyCourtJudgmentRule.assertCountyCourtJudgementCanBeRequested(claim, countyCourtJudgment.getCcjType());
 
+        if (claim.getResponse().isPresent()) {
+            Response response = claim.getResponse().orElseThrow(IllegalStateException::new);
+            if (claim.getState().equals(ClaimState.STAYED) && ResponseUtils.isAdmissionResponse(response)) {
+                claim = caseRepository.saveCaseEvent(authorisation, claim, LIFT_STAY);
+            }
+        }
```

Comment:
The throw is untestable - refactor `claim.getResponse()` to a variable and after verifying its presence just call `get()`

However, do we even need this logic? Isn't it enough that we want to save a judgment and the case is stayed for us to lift the stay?

Reply:
done for your first statement. 

We need this logic because changing state doesn't apply for default CCJ.

Reply:
I can do with just inner `if` but we don't need to check any of that for default CCJ's where there is no response.